ios/Podfile

```diff
@@ -0,0 +1,120 @@
+require_relative './PodfileHelpers.rb'
```

Comment:
I'm afraid of accidental whitespace changes to this file — and thus, `update podfile.lock checksum` commits on `master`. Could we implement some kind of prettifier (I know `rufo` supports this kind of file pretty well) that would ensure that committed `Podfile` is clean? (Before installing pods, before committing, as CI step?)

Reply:
I generally agree with you that we should prettify this file, but looks like `@prettier/plugin-ruby` is still not supported in VSCode prettier extension, also it doesn't provide too many options yet. I think it is something that we can try out later, once the prettier plugin will be supported by VSCode extension so we can apply changes on file save. Anyway, imho accidental whitespaces won't hurt too much, it will be still better than it is now after merging this PR 😉

Reply:
I'm not sure we should rely on a specific editor's prettier plugin to prettify the file.

Accidental whitespace makes another person's job more difficult, when he/she deletes an extra space from a line and suddenly the lockfile checksum changes. Wouldn't [`husky`](https://github.com/typicode/husky)+[`rufo`](https://github.com/ruby-formatter/rufo) (installed with nix like cocoapods) do the job?

Reply:
Yes, having a git hook would be the best option, but imho having more dependencies (adding rufo) doesn't make sense to me if there are no strong needs for it. We use prettier and I'd rather use it with that ruby plugin instead. Also, having it as a pre-commit hook can have some downsides: you modified Podfile and left a trailing whitespace, then you commit it, on pre-commit hook it automatically fixes the Podfile... but we also need to do `pod install` after all if it applied some changes, in order to have the correct checksum!

Reply:
Silent prettify wouldn't be a good option here — `rufo` would have to fail the commit phase if `Podfile` would be changed after beautifying.

Reply:
Since we don't write a lot of Ruby, I think we want to choose something very lightweight. That said, I don't foresee accidental whitespace changes being a big problem because if someone adds/removes extra whitespace, they were likely making other changes to this file anyway.